### PR TITLE
feat: added mixture experimentalist contribution

### PR DESCRIPTION
**Pull request by blinodelka**
# Description

I added a mixture experimentalist package to the dependencies and documentation:)
https://pypi.org/project/mixture-experimentalist/

https://github.com/blinodelka/mixture_experimental_strategies 

*If the issue is on Github, simply link to it using the '#' symbol; otherwise, provide a notion page link)*:
- resolves #(issue)

# Type of change
- **feat**: A new feature

# Features (Optional)
The Mixture Experimentalist identifies novel experimental conditions under which a hybrid of different experimental sampling strategies is used. This mixture can include any custom strategies such as falsification, novelty, crucial experimentation, uncertainty, elimination, aesthetic preferences, and arbitrary preferred/dispreferred regions of the space. The selection of conditions is based on a weighted sum of the scores obtained from these strategies.

# Questions (Optional)

# Remarks (Optional)

# Screenshots (Optional)

### CODE REVIEW

**Comment by younesStrittmatter** [comment]
Hi @blinodelka. This is super amazing, and I am very excited about your contribution!
Before we proceed, I made a pull request to the original repository to incorporate a few minor changes.

**Review by younesStrittmatter — APPROVED** [review]
This looks good to me. The repository itself looks good too.

**Comment by younesStrittmatter** [comment]
Thanks a lot @blinodelka! This looks good to me. You don't need to create a new pull request. All the changes are already in this one.

**Review by benwandrew — COMMENTED** [review]
looks good, just one question: per the directory structure in `src` (`src/autora/experimentalist/sampler/mixture_experimentalist`), the function specified in the `__init__.py` (`mixture_sample`), and the placement under "Samplers in the `mkdocs.yml`, it seems like this is a type of sampler; specifically, a mixture of samplers? if so, shouldn't the `pyproject.toml` have `experimentalist-sampler-mixture` instead of just `experimentalist-mixture` in the relevant places?

**Comment by younesStrittmatter** [comment]
@benwandrew you are right. It is a sampler and should be called like that

File: pyproject.toml [hunk 1]
```diff
@@ -73,6 +74,10 @@ experimentalist-falsification =[
   "autora-experimentalist-falsification"
 ]
```
Comment (benwandrew) [inline pyproject.toml:76]:
```suggestion
```

File: pyproject.toml [hunk 2]
```diff
@@ -73,6 +74,10 @@ experimentalist-falsification =[
...
 
+experimentalist-mixture =[
```
Comment (younesStrittmatter) [inline pyproject.toml:77]:
```suggestion
experimentalist-sampler-mixture =[
```

File: pyproject.toml [hunk 3]
```diff
@@ -53,6 +53,7 @@ all-experimentalists = [
   "autora[experimentalist-sampler-model-disagreement]",
   "autora[experimentalist-sampler-uncertainty]",
   "autora[experimentalist-falsification]",
+  "autora[experimentalist-mixture]",
```
Comment (younesStrittmatter) [inline pyproject.toml:56]:
```suggestion
  "autora[experimentalist-sampler-mixture]",
```

**Review by younesStrittmatter — CHANGES_REQUESTED** [review]

**Review by benwandrew — CHANGES_REQUESTED** [review]
also, from where has this package been pre-released? i see the PyPI page but the Marina's repo has nothing under releases. finally, will probably need to reflect the change to `experimentalist-sampler-mixture` were necessary in the repo (eg, https://github.com/blinodelka/mixture_experimental_strategies#:~:text=pip%20install%20%2DU%20autora%5B%22mixture_experimentalist%22%5D)

File: pyproject.toml (hunk 1 above)
Comment (blinodelka) [reply pyproject.toml:76]: Yes, thanks -- just updated the repo! It's my first time publishing a package, so I didn't know I had to reflect releases in the repo too.

**Comment by blinodelka** [comment]
@younesStrittmatter @benwandrew thanks for your help! Just updated the repository with the suggested name changes (all the changes are reflected in the new version of the package 1.0.0a3: https://pypi.org/project/mixture-experimentalist/)

**Comment by blinodelka** [comment]
Although I just noticed that I run onto an issue when trying to add a new release to the github repository on the website (I guess it has something to do with the fact that I've been working with PyPI through console?):( Will try to resolve this!

**Comment by younesStrittmatter** [comment]
@blinodelka The newest release is on Pypi, which is all that matters to us. Should I create a pull request to add the GitHub action for releases to your repository to make it easier in the future? I can also quickly walk you through the release process via GitHub action if you want to.

**Comment by blinodelka** [comment]
@younesStrittmatter thank you -- I'd really appreciate if you could walk me through the release process on github! I can jump on a call now or later today if you have time.

**Comment by younesStrittmatter** [comment]
Changed the mkdocs.yml file to include the code references. As soon as the changes in the pyproject.toml file are addressed, this is ready to be merged.

**Comment by benwandrew** [comment]
can we commit the suggestions and get this merged!?

**Review by younesStrittmatter — APPROVED** [review]
looks good!

**Review by benwandrew — APPROVED** [review]
agreed; looks good!

**Pull request merged**